### ISSUE TITLE
CSharp Style - Blank Line After Close Brace

### ISSUE THREAD

**Issue by TheMagnificent11** [dotnet/roslyn#61861]
**Brief description:**

Please add a style analyzer for adding a blank line after a closing brace, if the next line isn't another closing brace.

StyleCop has an analyzer for this: [SA1513](https://documentation.help/StyleCop/SA1513.html)

Potential name: `csharp_new_line_after_close_brace`

**Languages applicable:**

C#

**Code example that the analyzer should report:**

```cs
if (myInt > 1)
{
    ExecuteSomeFunction(myInt);
}

// Note: blank line before this is desired
if (myInt == 0)
{
    ExecuteSomeOtherFunction(myInt);
}
```

However, there should be no failure for the following because the closing brace is followed by another closing brace on the next line.
```cs
public static class IntExtensions
{
    public static bool IsPositive(this int someValue)
    {
        return someValue > 0;
    }
}
```

**Additional information:**

None

**Documentation requirements:**

When this analyzer is implemented, it must be documented by following the steps at [Documentation for IDE CodeStyle analyzers](https://github.com/dotnet/roslyn/blob/main/docs/contributing/Documentation%20for%20IDE%20CodeStyle%20analyzers.md).

**Comment by CyrusNajmabadi** [comment]
This is currently available with dotnet_style_allow_statement_immediately_after_block_experimental.

**Comment by TheMagnificent11** [comment]
@CyrusNajmabadi can you please provide an example of how I configure that in my `.editorconfig`?

**Comment by CyrusNajmabadi** [comment]
I'd expect you could just do: `dotnet_style_allow_statement_immediately_after_block_experimental=false`.  I'm not really that familiar with editorconfig though.

**Comment by TheMagnificent11** [comment]
Thanks, that works 😀
